docs/changes.txt

```diff
@@ -78,17 +87,17 @@ Thanks to Christian Zagrodnick for these changes.
 
 3.3.1 (12 Dec 2008)
 -------------------
-  
+
```

Comment:
These whitespace changes are really annoying. Why are they in this diff? If you must include them, please separate them out into their own commit.